v2/swagger.yaml

```diff
@@ -0,0 +1,629 @@
+      gh_prs_last_year_closed:
+        type: array
+        items:
+          type: integer
+      gh_issues_last_year_opened:
+        type: array
+        items:
+          type: integer
+      gh_prs_last_month_closed:
+        type: array
+        items:
+          type: integer
+      gh_prs_last_month_opened:
+        type: array
+        items:
+          type: integer
+      vertex_label:
+        type: array
+        items:
+          type: string
+      gh_contributors_count:
+        type: array
+        items:
+          type: integer
+      gh_issues_last_month_opened:
+        type: array
+        items:
+          type: integer
+      gh_issues_last_year_closed:
+        type: array
+        items:
+          type: integer
+      latest_version:
+        type: array
+        items:
+          type: integer
+      gh_open_issues_count:
+        type: array
+        items:
+          type: integer
+      name:
+        type: array
+        items:
+          type: string
+      package_relative_used:
+        type: array
+        items:
+          type: string
+      package_dependents_count:
+        type: array
+        items:
+          type: integer
+      libio_total_releases:
+        type: array
+        items:
+          type: string
+      tokens:
+        type: array
+        items:
+          type: string
+  ComponentAnalysesDataDetails:
+    title: Component Analyses Package and Version
+    description: Component analyses data for package and version
+    properties:
+      version:
+        $ref: "#/definitions/Version"
+      package:
+        $ref: "#/definitions/Package"
+  PremiumVulnerabilityFields:
+    title: PremiumVulnerabilityFields
+    description: Premium Vulnerability details, available only to registered
+    properties:
+      malicious:
+        type: boolean
+      patch_exists:
+        type: boolean
+      fixable:
+        type: boolean
+      exploit:
+        type: string
+      description:
+        type: string
+      fixed_in:
+        type: string
+      references:
+        type: array
+        items:
+          type: object
+          properties:
+            title:
+              type: string
+            url:
+              type: string
+  Vulnerability:
+    title: Vulnerability
+    description: Vulnerability details
+    properties:
+      cve_ids:
+        type: array
+        items:
+          type: string
+      cvss:
+        type: number
+        format: float
+      cvss_v3:
+        type: string
+      cwes:
+        type: array
+        items:
+          type: string
+      id:
+        type: string
+      severity:
+        type: string
+        enum:
+          - low
+          - medium
+          - high
+      title:
+        type: string
+      url:
+        type: string
+      premium_details:
+        description: Premium details about a Vulnerability, available only to registered users
+        $ref: "#/definitions/PremiumVulnerabilityFields"
+  CVE:
```

Comment:
If this indicates 'public vulnerability', we need appropriate name for this field

Reply:
This is used in Component Analysis, @deepak1725 will address it as part his spec update PR.